setup.cfg

```diff
@@ -5,7 +5,7 @@ description_file = README.md
 # B905 is for using zip without the `strict` argument, which was introduced in
 # python 3.10. We should probably add this check (remove it from the ignore) when we
 # require 3.10.
-ignore = W503, E203, N806, B905, B907
+ignore = W503, E203, N806, B905, B907, B028
```

Comment:
I think `B028` is the same as `B907`. It started out as B028 but then was converted to B907 in flake8-bugbear version 23.1.17 . We now require that version or newer, so I don't think we need `B028` in this list. See: https://github.com/PyCQA/flake8-bugbear#23117